server/sandbox_run_linux.go

```diff
@@ -763,7 +763,7 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 			}
 			g.AddMount(proc)
 		}
-		rootPair := s.defaultIDMappings.RootPair()
+		rootPair := sandboxIDMappings.RootPair()
```

Comment:
based on the title, I expected to see something like:
```suggestion
		rootPair := sandboxIDMappings.RootPair(); rootPair != nil {
		    rootPair = s.defaultIDMappings.RootPair()
		}
```

Reply:
the segfault came from s.defaultIDMappings being nil, rootPair is an object, not a pointer :)